scripts/qa_crowbarsetup.sh

```diff
@@ -2068,6 +2068,58 @@ function onadmin_get_ip_from_dhcp
         END{ if (res=="") exit 1; print res }' $leasefile
 }
 
+function onadmin_setup_nfs_server
+{
+    wait_for 150 10 "onadmin_get_ip_from_dhcp '$nfsservermac'" "node to get an IP from DHCP" "exit 78"
+    local nfs_server_node_ip=`onadmin_get_ip_from_dhcp "$nfsservermac"`
+
+    wait_for 150 10 "ping -q -c 1 -w 1 $nfs_server_node_ip >/dev/null" "ping to return from ${cloud}-lonelynode" "complain 82 'could not ping nfs-server VM ($nfs_server_node_ip)'"
+
+    # wait a bit for sshd.service on ${cloud}-lonelynode
+    wait_for 10 10 "ssh_password $nfs_server_node_ip 'echo'" "ssh to be running on ${cloud}-lonelynode" "complain 82 'sshd is not responding on ($nfs_server_node_ip)'"
+
+    local pubkey=`cat /root/.ssh/id_rsa.pub`
+    ssh_password $nfs_server_node_ip "mkdir -p /root/.ssh; echo '$pubkey' >> /root/.ssh/authorized_keys"
+
+    local pfile=`get_proposal_filename dns default`
+    crowbar dns proposal show default |
+        rubyjsonparse "
+            j['attributes']['dns']['records']['nfsserver']={};
+            j['attributes']['dns']['records']['nfsserver']['type']='A';
+            j['attributes']['dns']['records']['nfsserver']['values']=['$nfs_server_node_ip'];
+            puts JSON.pretty_generate(j)" > $pfile
+    crowbar dns proposal --file=$pfile edit default
+    rm -f $pfile
+    crowbar dns proposal commit default
+    local uri_base="http://${clouddata}${clouddata_base_path}"
+    local sle_pool=""
+    local sle_updates=""
+
+    case $(getcloudver) in
+        6)
+            sle_pool="$uri_base/SLES12-SP1-Pool/ sles12sp1"
+            sle_updates="$uri_base/SLES12-SP1-Updates/ sles12sp1up"
+        ;;
+        7)
+            sle_pool="$uri_base/$arch/SLES12-SP2-Pool/ sles12sp2"
+            sle_updates="$uri_base/$arch/SLES12-SP2-Updates/ sles12sp2up"
+        ;;
+    esac
+
+    inject="
+        set -x
+        zypper ar $sle_pool
+        zypper ar $sle_updates
+        zypper -n in nfs-kernel-server
```

Comment:
It seems indeed to be different between the SP1 and SP2 image. So I guess it's saver to keep it here.